clients/python/setup.py

```diff
@@ -31,10 +27,16 @@
 with open('README.rst') as f:
     readme = f.read()
 
+init = os.path.join(os.path.dirname(__file__), 'girder_client', '__init__.py')
+with open(init) as fd:
+    version = re.search(
+        r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
+        fd.read(), re.MULTILINE).group(1)
+
```

Comment:
This feels weird. Can we not simply import `girder_client` and extract `__version__` from it directly?